src/google-maps/map-ground-overlay/map-ground-overlay.ts

```diff
@@ -37,13 +38,19 @@ export class MapGroundOverlay implements OnInit, OnDestroy {
    */
   groundOverlay?: google.maps.GroundOverlay;
 
-  @Input() url!: string;  // Asserted in ngOnInit.
-
+  /** URL of the image that will be shown in the overlay. */
   @Input()
-  bounds!: google.maps.LatLngBounds|google.maps.LatLngBoundsLiteral;  // Asserted in ngOnInit.
+  set url(url: string) {
+    this._url.next(url);
+  }
 
-  @Input() clickable = false;
+  /** Bounds for the overlay. */
+  @Input() bounds: google.maps.LatLngBounds|google.maps.LatLngBoundsLiteral;
```

Comment:
Why not allow bounds to be changed in the same manner as url?

Reply:
I didn't know whether there wasn't some context behind it not being implemented in the first place.

Reply:
It's the same reason I didn't implement for url. I've tried to follow the Google Maps JavaScript API as closely as possible. Unlike most of the components, the Ground Overlay class does not have a way to alter the url or the bounds with their api once it's initialized, which is why I didn't implement them to begin with.

I think this is a good idea though, even if it doesn't exactly match the api, it improves the current functionality and I don't any see any downsides, but I think if we do it for url, then we should do it for bounds as well.

Reply:
I see. I gave it a shot, but it might be better to leave it as a follow-up PR. It seems like the Maps API throws an error if we pass in a `LatLngBoundsLiteral`, instead of a `LatLngBounds` and I couldn't find an API to convert between the two.

Reply:
Ok, no problem. Thanks for looking into that. LGTM